lynn/src/main/java/StackOfLinkedList.java

```diff
@@ -0,0 +1,83 @@
+import java.util.EmptyStackException;
+
+public class StackOfLinkedList<T> {
+
+    private Node<T> top; //top 노드를 가리키는 변수
+
+    private class Node<T> {
+        private T data;
+        private Node<T> next; //다음 노드를 가리킴
+
+        public Node(T data) {
+            this.data = data;
+        }
+    }
+
+    //가장 위에 있는 노드를 가져오고 삭제
+    public T pop() {
+        T item = peek(); //반환할 데이터
+        top = top.next; //기존 top의 다음 노드를 가리키도록 함
+        return item;
+    }
+
+    //새로운 노드 추가
+    public void push(T item) {
+        Node<T> newNode = new Node<T>(item); //새 노드 생성
```

Comment:
```suggestion
        Node<T> newNode = new Node<>(item);
```
과 같은 식으로 다이아몬드 오퍼레이터를 사용해도 좋을듯